src/php/lib/Grpc/BaseStub.php

```diff
@@ -38,9 +38,9 @@ class BaseStub
      *  - 'update_metadata': (optional) a callback function which takes in a
      * metadata array, and returns an updated metadata array
      *  - 'grpc.primary_user_agent': (optional) a user-agent string
-     * @param Channel $channel An already created Channel object (optional)
+     * @param Channel/InterceptorChannel $channel An already created Channel or InterceptorChannel object (optional)
```

Comment:
Does PHP documentation generator understand this construct?

Reply:
I think it can. The format is @param [Type] [name] [<description>]. Type and name are divided by space

Reply:
Does it have understanding of the slash in type name though? It might be better just to leave it "Channel" for descriptive purposes.

Reply:
Oh. I'll substitute it with '|'. This one can be used for sure and other people are using it like "int|string"